package-lock.json

```diff
@@ -55,7 +55,7 @@
         "inquirer": "^8.2.6",
         "lint-staged": "^14.0.1",
         "prettier": "3.0.3",
-        "storybook": "^7.4.1",
+        "storybook": "7.6.20",
```

Comment:
@damonmcc when you added the `^`s, did you change `package.json` manually? Asking because they don't show up here but they did for the `7.4` versions.

Reply:
oops i totally forgot about the lock file. other than manually changing `package.json` and then doing `npm install` to update package-lock.json, how would you modify both less manually?

i just did the above locally and I now see `^` in package-lock.json, but curious if there's a cleaner way

Reply:
just pushed changes

Reply:
What you did should be fine. Normally I'd say installing the individual packages with `npm i storybook@"^7.6.20"` should automatically update the lock accordingly, but that advice doesn't apply here if you're upgrading these with storybooks upgrade utility.